ci-operator/step-registry/upi/install/vsphere/upi-install-vsphere-commands.sh

```diff
@@ -137,6 +137,53 @@ oc wait --for=condition=Updated mcp/worker --timeout=30m
 
 }
 
+function deploy_ccm() {
+  set -x
+  echo "INFO" "Deploy External CCM....."
+  oc label --overwrite ns kube-system pod-security.kubernetes.io/enforce=privileged pod-security.kubernetes.io/audit=privileged pod-security.kubernetes.io/warn=privileged security.openshift.io/scc.podSecurityLabelSync=false
+  apiServerInternalURI="$(oc get infrastructure cluster -o jsonpath='{.status.apiServerInternalURI}{"\n"}')" || exit 3
+  apiServerInternalHost=${apiServerInternalURI#*//} 
+  apiServerInternalHost=${apiServerInternalHost%:*} 
+  set +x
+  source "${SHARED_DIR}/govc.sh"
+  echo ${GOVC_URL}
+  echo ${GOVC_DATACENTER}
+  sed -e "s/<vcenterhost>/${GOVC_URL}/g" -e "s/<vspheredatacenter>/${GOVC_DATACENTER}/g" -e "s/<vcenterusername>/${GOVC_USERNAME}/g" -e "/<vcenterpassword>/c \ \ ${GOVC_URL}.password: \"${GOVC_PASSWORD}\"" -e "s/<kubernetes_service_host>/${apiServerInternalHost}/g" "${SHARED_DIR}/vsphere-cloud-controller-manager.yaml" | oc create -f -
```

Comment:
I think we should move the GOVC string replacement to `upi-conf-vsphere-ccm-external` step.